src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -137,6 +137,16 @@ public static class CLIOptions
             JsonKey = "abort-test-on-fail"
         };
 
+        public static readonly CLIOption<bool> TestMultipleMutants = new CLIOption<bool>
+        {
+            ArgumentName = "--test-multiple-mutants",
```

Comment:
Maybe the name `--overlap-mutants` describes this feature better. As we already test multiple mutants at the same time. But this PR adds the ability to overlap mutants in the same testrun.

Reply:
I don't like the name _test-multiple-mutants_ either. But _overlap-mutants_ sounds strange to me.
Thinking about it, what about using reverse naming, such as _test-each-mutant-in-isolation_, or something similar.
Ultimately, we should test mutants in batches for performance reason, and this feature comes with no anticipated limitation (vs testing mutant one after the other).

Reply:
I think we should treat it as a temporary inverse feature flag and to give the feature a name we can live with like mix-mutations. So the flag would become `--disable-mix-mutations` or whatever other name we come up with. If we start naming features like these in such a way then we can also start removing them easily later. There shouldn't be any reason to turn this off unless it doesn't work.

I don't think the name is that important since I don't think we should plan to have this as an option for very long. As long as we have a good description people will find the flag if they need it.

Reply:
I propose `--disable-testing-mix-mutations`